README.md

```diff
@@ -174,7 +177,7 @@ gitHub live pages
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
+- [ ] **[Working menu bar]**
 - [ ] **[new_feature_2]**
 - [ ] **[new_feature_3]**
```

Comment:
- [ ] Kindly make your `README.md` file look professional by replacing the two placeholders under the `[Working menu bar]` with features you can implement in the future. Please implement it so that your **_README.md_** file looks better. 💻